### ISSUE TITLE
IP is blocked on school wifi

### ISSUE THREAD

**Issue by carsin** [elijaharita/dotsgame#12]
#11

**Comment by elijaharita** [comment]
qid i cant fix that

**Comment by elijaharita** [comment]
oh wait also its not open

**Comment by elijaharita** [comment]
now it is

**Comment by carsin** [comment]
help im bored

**Comment by elijaharita** [comment]
this might need a new issue
